### PR TITLE
T2: Number to words and roman numerals converter 

### PR DESCRIPTION
**Pull request by LogeshTMM**

### CODE REVIEW

File: T02/Program.cs [hunk 1]
```diff
@@ -0,0 +1,163 @@
+﻿// CONVERT THE GIVEN NUMBER INTO WORD FORMAT AND ROMAN NUMERALS FORMAT
+internal class Program {
+   static void Main (string[] args) {
+      Console.WriteLine ("Enter the number to convert into \"Words\" and \"Roman numerals\"");
+      if (!int.TryParse (Console.ReadLine (), out int num)) {
+         Console.WriteLine ("Invaild input");
+         return;
+      }
+      var numString = num.ToString ();
```
Comment (priyarathna) [inline T02/Program.cs:9]: num.ToString() is same as Console.ReadLine (). Use it.

File: T02/Program.cs [hunk 2]
```diff
@@ -0,0 +1,163 @@
+﻿// CONVERT THE GIVEN NUMBER INTO WORD FORMAT AND ROMAN NUMERALS FORMAT
```
Comment (priyarathna) [inline T02/Program.cs:1]: The output is right but the code is too complicated. Check [Karthick's](https://github.com/karthick-1205/Assignments/pull/35/files) for the simpler one.

**Review by priyarathna — CHANGES_REQUESTED** [review]

File: T02/Program.cs [hunk 3]
```diff
@@ -0,0 +1,174 @@
+﻿// CONVERT THE GIVEN NUMBER INTO WORD FORMAT AND ROMAN NUMERALS FORMAT
+internal class Program {
```
Comment (priyarathna) [inline T02/Program.cs:2]: Check column length and the output line is not relevant.

**Review by priyarathna — CHANGES_REQUESTED** [review]

File: T02/Program.cs [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+   static void PrintRoman (int[] arr, Dictionary<int, string> dic, string input) {
+      Console.Write ($"\nThe Roman numerals format for a given number is: ");
+      string temp = arr[2].ToString () + arr[3].ToString ();
+      if (arr[0] != 0) Console.Write (dic[arr[0] * 1000]);
+      if (arr[1] != 0) Console.Write (dic[arr[1] * 100]);
+      if (dic.TryGetValue (Convert.ToInt32 (temp), out string value)) {
+         Console.Write (value);
+         return;
+      } else if (arr[2] != 0 && arr[3] >= 1) Console.Write (dic[arr[2] * 10] + dic[arr[3]]);
```
Comment (priyarathna) [inline T02/Program.cs:46]: No else needed as return is used.

**Review by priyarathna — CHANGES_REQUESTED** [review]

File: T02/Program.cs [hunk 5]
```diff
@@ -40,10 +40,8 @@
       string temp = arr[2].ToString () + arr[3].ToString ();
       if (arr[0] != 0) Console.Write (dic[arr[0] * 1000]);
       if (arr[1] != 0) Console.Write (dic[arr[1] * 100]);
-      if (dic.TryGetValue (Convert.ToInt32 (temp), out string value)) {
-         Console.Write (value);
-         return;
-      } else if (arr[2] != 0 && arr[3] >= 1) Console.Write (dic[arr[2] * 10] + dic[arr[3]]);
+      if (dic.TryGetValue (Convert.ToInt32 (temp), out string value)) Console.Write (value);
```
Comment (priyarathna) [inline T02/Program.cs:43]: No need of 'return' here?

**Review by priyarathna — CHANGES_REQUESTED** [review]

File: T02/Program.cs (hunk 5 above)
Comment (LogeshTMM) [reply T02/Program.cs:43]: I didn't use it here.

**Review by priyarathna — APPROVED** [review]

**Pull request merged**